modules/rest_api/controllers/services/rest.php

```diff
@@ -1573,16 +1645,24 @@ private function esGetSpecialFieldLatLon(array $doc) {
    * @return string
    *   Formatted value.
    */
-  private function esGetSpecialFieldLon(array $doc) {
+  private function esGetSpecialFieldLon(array $doc, array $params) {
     // Check in case fields are in composite agg key.
     $root = isset($doc['key']) ? $doc['key'] : $doc['location'];
     if (empty($root['point'])) {
       return 'n/a';
     }
     $coords = explode(',', $root['point']);
-    $ew = $coords[1] >= 0 ? 'E' : 'W';
-    $lon = number_format(abs($coords[1]), 3);
-    return "$lon$ew";
+    $format = isset($params) ? $params[0] : "";
+    switch($format) {
```

Comment:
Another switch statement that can be replaced.

Reply:
Do you mean use if rather than switch?

Reply:
Sorry - ignore this comment.